docs/src/bootstrap.md

```diff
@@ -122,22 +130,14 @@ sum(ρs .≈ -1)
 
 Furthermore there are even a few cases where the estimate of the standard deviation of the random effect for the intercept is zero.
 ```@example Main
-sum(first.(first.(first.(σρ))) .≈ 0)
+σs = @where(df2, :type .== "σ", :group .== "subj", :names .== "(Intercept)").value 
+sum(σs .≈ 0)
 ```
 
 There is a general condition to check for singularity of an estimated covariance matrix or matrices in a bootstrap sample.
 The parameter optimized in the estimation is `θ`, the relative covariance parameter.
 Some of the elements of this parameter vector must be non-negative and, when one of these components is approximately zero, one of the covariance matrices will be singular.
 
-The boundary values are available as
-```@example Main
-samp2.m.optsum.lowerbd
-```
-so the check on singularity becomes
-```@example Main
-sum(θ -> any(θ .≈ samp2.m.optsum.lowerbd), samp2.θ)
-```
-
 The `issingular` method for a `LinearMixedModel` object that tests if a parameter vector `θ` corresponds to a boundary or singular fit.
 The default value of `θ` is that from the model but another value can be given as the second argument.
```

Comment:
I think I would omit this line since we don't have an example here and the practical use of this isn't obvious to a typical user.
```suggestion

```